test/promql/tests/data_plane/pipeline_overhead_test.yaml

```diff
@@ -0,0 +1,159 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.pipeline_alerts.yaml
+
+tests:
```

Comment:
We're using sum, but we're not testing any summing.

Reply:
I'm new to this prom unit testing, but from what I've learned so far, it is summing the 781 number of entries I have specified.  But I am probably missing some nuance which you are getting at @yftacherzog .... could you please elaborate by citing one of my input_series values, and why it is not summing?  thanks

Reply:
`sum` is a function that aggregates the **current value** across a vector of time series. It will not aggregate over time (this is `sum_over_time` which is aggregating over a range vector. e.g. my-metric[5m])

suppose that you have this:
foo{bar="a"}
1 1 1 1
foo{bar="b"}
1 1 1 1
foo{bar="c"}
1 1 1 1

`sum(foo)` will give us:
foo
3 3 3 3

So, suppose this is what we're after, then we need test cases in which we have more than one time series that fits the queries that we do to see that we're summing the way we expect.

Reply:
I think you should at least have more than one time series that should be **included** in the sum. one which should be excluded would be a nice addition.